### PR TITLE
NO-ISSUE: Do not merge - fixing broken CI jobs

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

File: deploy/operator/utils.sh [hunk 1]
```diff
@@ -70,8 +70,13 @@ function wait_for_condition() {
     echo "Waiting for (${object}) on namespace (${namespace}) with labels (${selector}) to be created..."
     for i in {1..40}; do
         oc get ${object} --selector="${selector}" --namespace=${namespace} |& grep -ivE "(no resources found|not found)" && break || sleep 10
+        oc get ${object} --namespace=${namespace} -o yaml
     done
 
+
+    echo "DEBUG: Loop ended"
+    oc get ${object} --namespace=${namespace} -o yaml
```
Comment (slaviered) [inline deploy/operator/utils.sh:78]: I think we should have this after line 81

**Comment by nmagnezi** [comment]
Filed https://issues.redhat.com/browse/OCPBUGS-511

**Pull request closed without merging**